src/components/Header/Header.js

```diff
@@ -1,36 +1,12 @@
 import React from 'react';
-import ItemLIst from '../ItemList/ItemLIst';
-import './Header.css';
+import ItemList from '../ItemList/ItemList';
 
-class Header extends React.Component {
+import './Header.css';
 
-  render() {
-    const menuItems = [
-      {
-        name: "Home",
-        id:''
-      },
-      {
-        name: "About Us",
-        id:''
-      },
-      {
-        name: "Services",
-        id:''
-      },
-      {
-        name: "Contacts",
-        id:''
-      }
+function Header(props){
+  return <ItemList />
```

Comment:
the previous version of Header was better because you have a reusable component with items as props. You can use it in footer for example. 
``` 
 <nav className="header-menu" >
      <ItemLIst items={menuItems}/>
 </nav>
```